src/router/index.ts

```diff
@@ -23,7 +23,7 @@ if (config.auth) {
 }
 // TODO secure with scope
 v1.use('/db', db);
-v1.get('/version/latest', (_req, res) => res.send(config.modules.latestVersion));
-v1.get('/version/oldest', (_req, res) => res.send(config.modules.oldestVersion));
+v1.get('/version/latest', (_req, res) => res.send(config.version));
+v1.get('/version/oldest', (_req, res) => res.send(config.version));
```

Comment:
since the dashboard would need its own refactor to work with this new approach, does make sense to remove this duplication and just call `/version`?

Reply:
Yes, but I figured that could be a follow-up PR paired with a dashboard change. Running this code locally with a local dashboard works with this change here.